src/Home.js

```diff
@@ -107,4 +110,21 @@ const styles = StyleSheet.create({
   selectedCheckboxText: {
     color: "white"
   },
+  buttonText: {
```

Comment:
Nit: prefer more explicit `addButtonText` (and `addButtonContainer`)

Reply:
I am using the same styling for the Save button on the Modal. So, I think buttonText is more suitable as it can be used at multiple places.